pkg/controller/envoy/ads_event.go

```diff
@@ -241,6 +256,7 @@ func (svc *ServiceEvent) handleRdsResponse(rsp *service_discovery_v3.DiscoveryRe
 		if err = anypb.UnmarshalTo(resource, routeConfiguration, proto.UnmarshalOptions{}); err != nil {
 			continue
 		}
+		svc.ack.ResourceNames = append(svc.ack.ResourceNames, routeConfiguration.GetName())
```

Comment:
also keep consistent with eds, check if the rds resource still need to subscribe, otherwise donot append

Reply:
fixed